recipes/reaper/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+cd src
+make
+mkdir -p $PREFIX/bin
+mv reaper $PREFIX/bin
+mv tally $PREFIX/bin
```

Comment:
Should tally, swan and minion be in a separate repository?

Reply:
Thats a good point. They do not depend on each other, but can work together (especially reaper and tally, when it comes to trim paired-end sequencing files). The website states that `Tally and Minion are companion programs and shipped alongside reaper`.
How is this solved in other software tool sets? What would be the benefit to have them separately?